src/card.css

```diff
@@ -0,0 +1,120 @@
+@font-face {
+  font-family: "Roboto";
+  src: url("./fonts/Roboto-Regular.ttf");
+  font-style: normal;
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: "Roboto";
+  src: url("./fonts/Roboto-Medium.ttf");
+  font-style: medium;
+  font-weight: 500;
+}
+
+@font-face {
+  font-family: "Roboto";
+  src: url("./fonts/Roboto-Bold.ttf");
+  font-style: bold;
+  font-weight: 700;
+}
+
+.card {
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+
+  padding-top: 32px;
+  padding-bottom: 16px;
+}
+
+.card * {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  color: #060b35;
+}
+
+.card__img {
+  margin-bottom: 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  margin-bottom: 4px;
+  width: 166px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card__pcode {
+  width: 166px;
+  margin-bottom: 16px;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
```

Comment:
Move repeatable values to variables, please

Reply:
@SergiyVS 
Don't leave unresolved comments, please